README.md

```diff
@@ -35,10 +35,11 @@ make docker
 - [Snap](https://snapcraft.io/edgexfoundry)
 
 **Run device-rfid-llrp**
-- [Run via Snap](#snap-development-and-testing)
+- Snap
+    - For the snap, refer to the [snap](snap) directory.
 
 - Docker
-    - Use [compose-builder](https://github.com/edgexfoundry/edgex-compose/tree/jakarta/compose-builder)
+    - Use [compose-builder](https://github.com/edgexfoundry/edgex-compose/tree/main/compose-builder)
```

Comment:
We are release this for Jakarta, so this needs to stay as is.

Reply:
fixed.